test/features/contributor/presentation/cubit/contributor_cubit_test.dart

```diff
@@ -1,49 +1,62 @@
 import 'package:bloc_test/bloc_test.dart';
-import 'package:coffeecard/features/contributor/data/datasources/contributor_repository.dart';
-import 'package:coffeecard/features/contributor/domain/entities/contributor.dart';
+import 'package:coffeecard/core/errors/failures.dart';
+import 'package:coffeecard/features/occupation/domain/usecases/get_occupations.dart';
+import 'package:coffeecard/features/occupation/presentation/cubit/occupation_cubit.dart';
+import 'package:coffeecard/features/contributor/domain/usecases/fetch_contributors.dart';
 import 'package:coffeecard/features/contributor/presentation/cubit/contributor_cubit.dart';
+import 'package:dartz/dartz.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:mockito/annotations.dart';
 import 'package:mockito/mockito.dart';
+import 'package:coffeecard/features/contributor/domain/entities/contributor.dart';
 
 import 'contributor_cubit_test.mocks.dart';
 
-const dummyContributors = [
-  Contributor(
-    name: 'name',
-    avatarUrl: 'avatarUrl',
-    githubUrl: 'githubUrl',
-  ),
-];
-
-@GenerateMocks([ContributorRepository])
+@GenerateMocks([FetchContributors])
 void main() {
-  group('contributor cubit tests', () {
-    late ContributorCubit contributorCubit;
-    final repo = MockContributorRepository();
+  late ContributorCubit cubit;
+  late MockFetchContributors fetchContributors;
 
-    setUp(() {
-      contributorCubit = ContributorCubit(repository: repo);
-    });
+  setUp(() {
+    fetchContributors = MockFetchContributors();
+    cubit = ContributorCubit(fetchContributors: fetchContributors);
+  });
 
-    test('initial state is ContributorLoaded', () {
-      expect(contributorCubit.state, const ContributorLoaded([]));
-    });
+  test('initial state is ContributorInitial', () {
+    expect(cubit.state, const ContributorInitial());
+  });
 
-    blocTest<ContributorCubit, ContributorState>(
-      'getContributors emits [Loaded] when the repo returns a list of contributors',
-      build: () {
-        when(repo.getContributors()).thenAnswer((_) => dummyContributors);
-        return contributorCubit;
-      },
-      act: (cubit) => cubit.getContributors(),
-      expect: () => [
-        const ContributorLoaded(dummyContributors),
-      ],
-    );
+  group(
+    'getContributors',
+    () {
+      const tContributors = [
```

Comment:
Don't use abbreviations when naming variables
```suggestion
      const testContributors = [
```